app/test/e2e/favourite/favourite-delete.spec.js

```diff
@@ -0,0 +1,90 @@
+const nock = require('nock');
+const chai = require('chai');
+const mongoose = require('mongoose');
+const Favourite = require('models/favourite.model');
+const { USERS } = require('../utils/test.constants');
+const {
+    createFavourite,
+    mockGetUserFromToken,
+    ensureCorrectError,
+    mockDeleteFavouriteResourceFromGraph
+} = require('../utils/helpers');
+
+const { getTestServer } = require('../utils/test-server');
+
+chai.should();
+
+let requester;
+
+nock.disableNetConnect();
+nock.enableNetConnect(process.env.HOST_IP);
+
+describe('Delete favourites', () => {
+    beforeEach(async () => {
+        if (process.env.NODE_ENV !== 'test') {
+            throw Error(`Running the test suite with NODE_ENV ${process.env.NODE_ENV} may result in permanent data loss. Please use NODE_ENV=test.`);
+        }
+
+        requester = await getTestServer();
+
+        await Favourite.deleteMany({}).exec();
+    });
+
+    it('Deleting a favourite while not being authenticated should return a 401', async () => {
+        const favourite = await (new Favourite(createFavourite({ userId: USERS.USER.id }))).save();
+
+        const response = await requester
+            .delete(`/api/v1/favourite/${favourite._id}`);
+
+        ensureCorrectError(response.body, 'Unauthorized');
```

Comment:
are these validating the http response code? I think you have a few of those missing